apps/www/components/Realtime/examples/chat-example.tsx

```diff
@@ -0,0 +1,191 @@
+'use client'
+
+import { useState } from 'react'
+import type { ExampleLayoutProps } from '../example-layout'
+
+const instanceId = Math.random().toString(36).substring(2, 9)
+
+const appJsCode = `import { useEffect, useState, useRef } from 'react';
+import './styles.css';
+import { createClient } from '@supabase/supabase-js';
+
+// Initialize Supabase client
+const supabaseUrl = '${process.env.NEXT_PUBLIC_SUPABASE_REALTIME_URL}';
```

Comment:
all these likely needs to be changed to `NEXT_PUBLIC_SUPABASE_URL` and `NEXT_PUBLIC_SUPABASE_ANON_KEY` so that we consolidate all the www stuff into one project